demos/text_to_speech_demo/python/models/mel2wave_ie.py

```diff
@@ -151,7 +159,7 @@ def forward_rnn(self, mels, upsampled_mels, aux):
 
         active_network = self.batch_sizes.index(b_size)
 
-        h1, h2, x = self.get_rnn_init_states(b_size, self.rnn_width)
+        h1, h2, x = self.get_rnn_init_states(b_size, 512)
```

Comment:
h1 and h2 inputs have shapes in model: `[1, 512]`. But `self.rnn_width` value is equal 1. In old api this working (when h1 and h2 have shape `[1, 1]`, but in model `[1,512]`), but in new api it is not working. I changed initial definition value for `rnn_width`

Reply:
Okay. Then you might also remove `self.rnn_width`, since it is not used anymore

Reply:
I use it variable, but changed definition: `self.rnn_width = self.rnn_model.input('h1.1').shape[1]`. Maybe it makes sense to use it to avoid magic constants `512`.

Reply:
I didn't noticed the update, sorry, now is fine :)